nextcloudappstore/core/models.py

```diff
@@ -20,6 +21,19 @@ def search(self, terms, lang):
         query = reduce(lambda x, y: x & y, predicates, Q())
         return queryset.filter(query)
 
+    def get_compatible(self, platform_version):
```

Comment:
We also need to think about what do with releases that depend on a minor platform version, e.g.

```xml
<owncloud min-version="9.0.1" max-version="9.0"/>
```
This release should appear under the 9.0 tab in the web interface, but in the API we get the full current platform version (e.g. 9.0.2) and therefore should not be included in the result.